src/libtriton/includes/triton/armOperandProperties.hpp

```diff
@@ -57,6 +57,8 @@ namespace triton {
           //! The extend size (in bits).
           triton::uint32 extendSize;
 
+          //! Subtracted flag.
```

Comment:
Maybe a short comment about what is this `subtracted` flag and what is used for.

Reply:
A quick note about this flag and its name. In the case of a memory access, there's the possibility to add **and subtract** an immediate or a register to the base register to compute the memory address (for instance, `LDR<c> <Rt>, [<Rn>{, #+/-<imm12>}]` and `LDR<c> <Rt>, [<Rn>,+/-<Rm>{, <shift>}]{!}`). This is encoded in a bit usually named `U`. Here, I'm using the same name as Capstone use for consistency and simplicity.

An extra fact on this flag: the use of the `U` bit (or at least, how Capstone reports it) is not "consistent" (depending on instruction mode and memory access type). There are cases where it is used and other were it is not. Here are some examples (notice the value of the immediate and whether the `Substracted` flag is present or not):

```shell
cstool -d arm "\x04\x00\x11\xe4" 1000
1000  04 00 11 e4  ldr  r0, [r1], #-4
    op_count: 3
        operands[0].type: REG = r0
        operands[0].access: WRITE
        operands[1].type: MEM
            operands[1].mem.base: REG = r1
        operands[1].access: READ
        operands[2].type: IMM = 0x4
        Subtracted: True
    Write-back: True
    Registers read: r1
    Registers modified: r0 r1
    Groups: arm
```

```shell
cstool -d thumb "\x51\xf8\x04\x09" 1000
1000  51 f8 04 09  ldr  r0, [r1], #-4
    op_count: 3
        operands[0].type: REG = r0
        operands[0].access: WRITE
        operands[1].type: MEM
            operands[1].mem.base: REG = r1
        operands[2].type: IMM = 0xfffffffc
    Write-back: True
    Registers read: r1
    Registers modified: r0 r1
    Groups: thumb2
```

```shell
$ cstool -d arm "\x04\x00\x31\xe5" 1000
1000  04 00 31 e5  ldr  r0, [r1, #-4]!
    op_count: 2
        operands[0].type: REG = r0
        operands[0].access: WRITE
        operands[1].type: MEM
            operands[1].mem.base: REG = r1
            operands[1].mem.disp: 0xfffffffc
        operands[1].access: READ
    Write-back: True
    Registers read: r1
    Registers modified: r0 r1
    Groups: arm
```

Reply:
Commit 6b44a7817cbde713c2a89a4293375b902b8e9360 adds a comment to better explain the flag.